Core/Extensions/InteropExtensions.cs

```diff
@@ -54,6 +54,24 @@ public static unsafe Span<T> AsSpan<T>(this IntPtr ptr, int length, int prefixBy
 		}
 #endif
 
+		/// <summary>Returns the pointer.</summary>
+		/// <typeparam name="T">The type of items.</typeparam>
+		/// <param name="ptr">A pointer to the starting address of a specified number of <typeparamref name="T"/> elements in memory.</param>
+		/// <param name="length">The number of <typeparamref name="T"/> elements to be included in the pointer.</param>
+		/// <param name="prefixBytes">Bytes to skip before starting the span.</param>
+		/// <param name="allocatedBytes">If known, the total number of bytes allocated to the native memory in <paramref name="ptr"/>.</param>
+		/// <returns>A pointer that represents the memory.</returns>
+		/// <exception cref="System.InsufficientMemoryException"></exception>
+		public static unsafe T* AsPointer<T>(this IntPtr ptr, int length, int prefixBytes = 0, SizeT allocatedBytes = default) where T : unmanaged
```

Comment:
Will you rename this to `AsUnmanagedArrayPointer`?

Reply:
Sure.